platform/files.go

```diff
@@ -20,10 +20,10 @@ import (
 // analyzed.toml
 
 type AnalyzedMetadata struct {
-	PreviousImage *ImageIdentifier `toml:"image"`
-	Metadata      LayersMetadata   `toml:"metadata"`
-	RunImage      *ImageIdentifier `toml:"run-image,omitempty"`
-	BuildImage    *ImageIdentifier `toml:"build-image,omitempty"`
+	PreviousImageRef string           `toml:"image-ref"`
```

Comment:
i think this is resolved with the more recent changes and the file is now backwards compatible.

Reply:
Even though the information is preserved, this TOML annotation will render the file as:
```
image-ref: "previous-image-ref"
```
 instead of 
```
[image]
reference = "previous-image-ref"
```
as required by the platform spec [here](https://github.com/buildpacks/spec/blob/main/platform.md#analyzedtoml-toml).

Reply:
I think i'm [shimming it appropriately for older apis](https://github.com/buildpacks/lifecycle/pull/994/files#diff-631f47fe759684bf2670a0a309f32e09dbcf15a1e470097b7e9ee15f0fda3d92R93) ... can we change the spec?  😁

Reply:
oic... sorry, I missed that!

Reply:
We can, for newer platforms, but we'd need to include it in the [spec PR](https://github.com/buildpacks/spec/pull/335) with a justification for why it needs to be different. Personally I don't think our hatred of pointers is sufficient to change something that people are already used to.